src/Samples/Toolkit.SampleApp.Maui/Samples/PopupViewerSample.xaml

```diff
@@ -10,9 +10,9 @@
 
 
             <Border Background="#99ffffff" x:Name="popupPanel" IsVisible="False">
-                <Border HorizontalOptions="Center" VerticalOptions="Center" Background="{AppThemeBinding Dark=Black, Light=White}" Margin="0,50" Padding="20">
+                <Border HorizontalOptions="Center" VerticalOptions="Center" Background="{AppThemeBinding Dark=Black, Light=White}" Margin="0,32" Padding="12">
                     <Grid>
-                        <esriTK:PopupViewer x:Name="popupViewer" Padding="20" MaximumWidthRequest="400" HeightRequest="400" PopupAttachmentClicked="popupViewer_PopupAttachmentClicked" />
+                        <esriTK:PopupViewer x:Name="popupViewer" Padding="12" MaximumWidthRequest="400" MaximumHeightRequest="400" PopupAttachmentClicked="popupViewer_PopupAttachmentClicked" />
```

Comment:
Did you try this on all platforms? I seem to recall there's a reason we're using HeightRequest and not MaximumHeightRequest

Reply:
I now tested on all platforms, and the updated layout works well everywhere.